Automattic/jetpack
# Jetpack Social: Add share meter

This PR adds a share meter to the header of the Jetpack Social admin page, displaying to the user how many shares they have left.

#### Changes proposed in this Pull Request:

* Add shares meter to Jetpack Social admin page.
* Improve buttons in the header.

#### Other information:


#### Jetpack product discussion


1202436584533778-as-1202583289493859/f

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Make sure the share meter displays properly on the admin page.
* Publicize a new post.
* Make sure the meter goes up.
